### PR TITLE
more doxygen: fortran files starting with the letter 'c'

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of https://github.com/NOAA-EMC/NCEPLIBS-bufr/issues/246

Some more documentation fixes.

This PR fixes all doxygen warnings for fortran files starting with the letter 'c'.

### CODE REVIEW

**Review by jbathegit — APPROVED** [review]

File: src/cmsgini.f [hunk 1]
```diff
@@ -16,20 +15,22 @@
...
-C> @param[inout] NBYT - integer: actual length (in bytes) of "compressed data portion" of section 4
-C> (i.e. all of section 4 except for the first four bytes).
+C> @param[in] LUN - integer: I/O stream index into internal memory arrays.
+C> @param[out] MESG - integer(*): BUFR message.
+C> @param[in] SUBSET - character*8: Table A mnemonic for type of BUFR message being written.
+C> @param[in] IDATE - integer: date-time stored within Section 1 of BUFR message being written,
+C>                    in format of either YYMMDDHH or YYYYMMDDHH, depending on datelen() value.
+C> @param[in] NSUB - integer: number of subsetsa in MESG
+C> @param[inout] NBYT - integer:
+C>                      - On input, contains the length (in bytes) of Section 4, except for
```
Comment (edwardhartnett) [inline src/cmsgini.f:27]: Why are you adding all these blank spaces?

Are you counting on other developers maintaining your spacebar-based formatting moving forward?

We are not doing spacebar-based formatting in any other NCEPLIBS codes. We leave one blank space and let doxygen work out the foramatting.

File: src/cktaba.f [hunk 2]
```diff
@@ -1,56 +1,39 @@
...
-C> @param[in] LUN - integer: i/o stream index into internal memory arrays.
-C> @param[out] SUBSET - character*8: table a mnemonic for type of bufr message
-C> being checked:
-C> "        " = IRET equal to 11 (see IRET below) and not using Section 3 decoding.
-C> @param[out] JDATE    - integer: date-time stored within section 1 of bufr
-c> message being checked, in format of either yymmddhh or
-c> yyyymmddhh, depending on datelen() value.
+C> @param[in] LUN - integer: I/O stream index into internal memory arrays.
+C> @param[out] SUBSET - character*8: Table A mnemonic
+C>                      - "        " = IRET equal to 11 (see below) and not
```
Comment (edwardhartnett) [inline src/cktaba.f:25]: We don't use '=' in a bullet list. We just provlide the value and the meaning, like this:
* 12 means something.
* -12 means something else.

File: src/cmsgini.f (hunk 1 above)
Comment (jbathegit) [reply src/cmsgini.f:27]: I realize this isn't needed for doxygen, but it's very helpful for me as a developer (for readability purposes) when I'm working inside of the source code itself.

File: src/cktaba.f (hunk 2 above)
Comment (jbathegit) [reply src/cktaba.f:25]: ok, I can get that changed

**Comment by jbathegit** [comment]
Not sure why the MacOS build keeps crashing.  The output means something about needing a "brew upgrade"!?

Screenshot 2023-01-19 105742

then later on:

Screenshot 2023-01-19 105822

**Comment by jbathegit** [comment]
OK, so it just let me rerun the checks and merge #267, so I guess I'll re-merge that into this branch and try (yet) again (?)

**Comment by jbathegit** [comment]
So the MacOS build is now fine, but the Linux checks are now failing in the python build.  WTF!?

**Comment by jbathegit** [comment]
And now it worked - go figure!

**Comment by edwardhartnett** [comment]
OK we should take out brew update from the mac build. I will submit a PR. It's not needed and often fails...

**Pull request merged**